terraform/testcases/standard_otlp_metric_trace/otconfig.tpl

```diff
@@ -0,0 +1,31 @@
+extensions:
+  pprof:
+    endpoint: 0.0.0.0:1777
```

Comment:
Why do we need pprof?

Reply:
I believe pprof was added because Alex was building the otconfig based on the otlp_trace and otlp_metric testcases which include pprof.  pprof can be used to collector performance profiles and investigate issues with the Collector but if we decide it is not necessary then I can remove it.

Reply:
My apologies for the late reply. But yes, Huy is correct.

Reply:
Is it necessary?

Reply:
I found this [note](https://github.com/humivo/aws-otel-test-framework#:~:text=Note%3A%20Please%20enable%20pprof%20extension%20in%20your%20otconfig.tpl%20for%20the%20debugging%20purpose.) in the test framework README advising to enable pprof extension